File: 5_multiple_inheritance.py
class Company:
    def __init__(self, name, id) -> None:
        self.name = name
        self.id = id

    def display(self):
        print("Employee Name:", self.name, "and", "Employee Id:", self.id)

class Department:
    def __init__(self, department_name) -> None:
        self.department_name = department_name

    def display_department(self):
        print("Department:", self.department_name)

class Employee(Company, Department):
    def __init__(self, name, id, department_name) -> None:
        Company.__init__(self, name, id) 
        Department.__init__(self, department_name)  


obj = Employee("Gray", 23143, "Sales")
obj.display()         
obj.display_department() 

File: 5_multiple_inheritance.cpp
#include <bits/stdc++.h>
using namespace std;

class Company {
public:
    string name;
    int id;

    Company(string name, int id) {
        this->name = name;
        this->id = id;
    }

    void display() {
        cout << "Employee Name: " << name << " and Employee Id: " << id << endl;
    }
};

class Employee : public Company {
public:
    Employee(string name, int id)
        : Company(name, id) {}  
};

int main() {
    Employee obj("Gray", 23143);
    obj.display();

    return 0;
}